runtime/doc/options.txt

```diff
@@ -2441,7 +2441,14 @@ A jump table for the options with a short description can be found at |Q_op|.
 	  item		default		Used for ~
 	  stl:c		' ' or '^'	statusline of the current window
 	  stlnc:c	' ' or '='	statusline of the non-current windows
+	  horiz:c	'─' or '-'      horizontal separators |:split|
```

Comment:
perhaps mention somewhere around here that most of these only take effect when `laststatus=3`

Reply:
Is it fine now after the latest commit?